architecture/additional_concepts/authorization.adoc

```diff
@@ -48,46 +48,225 @@ endif::[]
+  subjectrulesreviews						[]			[]		[create]
+  subjectrulesreviews.authorization.openshift.io		[]			[]		[create]
+  templateconfigs						[]			[]		[create delete deletecollection get list patch update watch]
+  templateconfigs.template.openshift.io				[]			[]		[create delete deletecollection get list patch update watch]
+  templateinstances						[]			[]		[create delete deletecollection get list patch update watch]
+  templateinstances.template.openshift.io			[]			[]		[create delete deletecollection get list patch update watch]
+  templates							[]			[]		[create delete deletecollection get list patch update watch]
+  templates.template.openshift.io				[]			[]		[create delete deletecollection get list patch update watch]
+
+
+Name:		basic-user
+Labels:		<none>
+Annotations:	openshift.io/description=A user that can get basic information about projects.
+		rbac.authorization.kubernetes.io/autoupdate=true
+PolicyRule:
+  Resources						Non-Resource URLs	Resource Names	Verbs
+  ---------						-----------------	--------------	-----
+  clusterroles						[]			[]		[get list]
+  clusterroles.authorization.openshift.io		[]			[]		[get list]
+  clusterroles.rbac.authorization.k8s.io		[]			[]		[get list watch]
+  projectrequests					[]			[]		[list]
+  projectrequests.project.openshift.io			[]			[]		[list]
+  projects						[]			[]		[list watch]
+  projects.project.openshift.io				[]			[]		[list watch]
+  selfsubjectaccessreviews.authorization.k8s.io		[]			[]		[create]
+  selfsubjectrulesreviews				[]			[]		[create]
+  selfsubjectrulesreviews.authorization.openshift.io	[]			[]		[create]
+  storageclasses.storage.k8s.io				[]			[]		[get list]
+  users							[]			[~]		[get]
+  users.user.openshift.io				[]			[~]		[get]
 ----
 ====
 
-The following excerpt from viewing policy bindings shows the above roles bound
+The following excerpt from viewing local role bindings shows the above roles bound
 to various users and groups:
 
 ====
 
 [options="nowrap"]
 ----
-RoleBinding[admins]:
-				Role:	admin
-				Users:	[alice system:admin]
-				Groups:	[]
-RoleBinding[basic-user]:
-				Role:	basic-user
-				Users:	[joe]
-				Groups:	[devel]
+oc describe rolebinding.rbac admin basic-user -n alice-project
+----
+
+----
+Name:		admin
+Labels:		<none>
+Annotations:	<none>
+Role:
+  Kind:	ClusterRole
+  Name:	admin
+Subjects:
+  Kind	Name		Namespace
+  ----	----		---------
+  User	system:admin
+  User	alice
+
+
+Name:		basic-user
+Labels:		<none>
+Annotations:	<none>
+Role:
+  Kind:	ClusterRole
+  Name:	basic-user
+Subjects:
+  Kind	Name	Namespace
+  ----	----	---------
+  User	joe
+  Group	devel
 ----
 ====
 
-The relationships between the the policy roles, policy bindings, users, and
+The relationships between roles, role bindings, users, and
 developers are illustrated below.
 
-image::authorization.png[{product-title} Authorization Policy]
+image::authorization.png[{product-title} RBAC]  // TODO this image needs to be updated
```

Comment:
@ahardin-rh You created the original image, so I can work with you to try to get it updated.

Reply:
@enj Thanks! Let's sync up to determine what needs changed in the image and I will coordinate with our graphics team from Brand to get this updated.